rllib/models/modelv2.py

```diff
@@ -223,6 +232,25 @@ def from_batch(self, train_batch, is_training=True):
             i += 1
         return self.__call__(input_dict, states, train_batch.get("seq_lens"))
 
+    def get_view_requirements(self, is_training=False):
+        """Returns a list of ViewRequirements for this Model (or None).
+
+        A ViewRequirement object tells the caller of this Model, which
+        data at which timesteps are needed by this Model. This could be a
+        sequence of past observations, internal-states, previous rewards, or
+        other episode data/previous model outputs.
+
+        Args:
+            is_training (bool): Whether the returned requirements are for
+                training or inference (default).
+
+        Returns:
+            Optional[List[ViewRequirement]]: The list of ViewRequirement
+                specs. None for standard/backward-compatible behavior
+                (aka input_dict).
```

Comment:
Can we split these miscellaneous cleanups / refactorings into another PR? It's hard to review a large PR when they are mixed in.